accounts/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.auth.admin import UserAdmin
+
+from .forms import CustomUserChangeForm, CustomUserCreationForm
+
+CustomUser = get_user_model()
+
+
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+    model = CustomUser
+    list_display = [
+        "email",
+        "username",
+        "is_superuser",
+    ]
+
+
```

Comment:
There is a few extra empty lines in these `py` files just wondering if thats needed or can we format these files.

Reply:
Its a part of the linting, python expects a newline at eof